doc/src/guide/listeners.asciidoc

```diff
@@ -203,6 +203,23 @@ ranch:set_max_connections(tcp_echo, MaxConns).
 
 The change will occur immediately.
 
+=== Running out of file descriptors
+
+Operating systems have limits on the number of sockets
+which can be opened by applications. When this maximum is
+reached the listener can no longer accept new connections. The
+accept rate of the listener will be automatically reduced, and a
+warning message will be logged.
+
+[source]
```

Comment:
Doubt that works if you do "make docs", does it? Either way, better to put ---- before/after instead of source.